docs/data-type-support.rst

```diff
@@ -0,0 +1,75 @@
+.. meta::
+   :description: hipcub API library data type support
+   :keywords: hipcub, ROCm, API library, API reference, data type, support
+
+.. _data-type-support:
+
+******************************************
+Data type support
+******************************************
+
+* Supported input and output types.
+
+  .. list-table:: Supported Input/Output Types
+    :header-rows: 1
+    :name: supported-input-output-types
+
+    *
+      - Input/Output Types
+      - Library Data Type
+      - AMD Support
+      - CUDA Support
+    *
+      - int8
+      - int8_t
```

Comment:
If we rely on the landing page explaining the data types, then this column is obsolete

Reply:
fixed